src/plone/api/tests/base.py

```diff
@@ -58,6 +58,10 @@ def tearDownZope(self, app):
 
 FIXTURE = PloneApiLayer()
 INTEGRATION_TESTING = IntegrationTesting(
-    bases=(FIXTURE,), name='PloneApiLayer:Integration')
+    bases=(FIXTURE, ),
```

Comment:
@hvelarde seems we don't agree on the style, that's a hard no go for me (at least on my projects).

As soon as a statement can not fit into a single line, then everything needs to be on their own, mostly for readability. This guy explains it way better than this short comments of mine: http://treyhunner.com/2017/07/craft-your-python-like-poetry/ and I wholeheartedly agree with him 😄